trimmer_decorator.rb

```diff
@@ -0,0 +1,10 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  MAXIMUM_NAME_LENGTH = 10
+
+  def correct_name
+    # name = @nameable.correct_name
```

Comment:
- [ ] Kindly remove this line of commented code as it can make your code messy and hard to read